app/views/comments/_form.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= form_with model: Comment, url: user_post_comments_path(@user, @post) do |form| %>
+  <div class="form-control">
+    <%= form.label :text, "Text" %>
+    <%= form.text_area :text, size: "70x15" %>
+  </div>
+  <div class="form-submit">
+    <%= form.submit "Add Comment" %>
+  </div>
+<% end %>
```

Comment:
- [ ] There should be a form in the `views/comments/new.html.erb` file which accepts input for creating comments. This is missing in your folder structure. I am sure this is because you did not have the `new` action in your comments controller.

Reply:
The task requires that we create a form to handle the creation of comments. It did not specifically ask that we create a view for it. So I put the form in the show view of the post controller. It will make it easy for the user to comment on a post easily without having to navigate to another page.